app/controllers/webhooks/github_controller.rb

```diff
@@ -33,14 +33,17 @@ def callback
 
   def push_event(chat, body)
     if body['commits'].length > 1
-      msg = '%s pushed %d commits %s'
+      msg = '%s pushed %d commits to %s %s'
```

Comment:
FYI, http://guides.rubyonrails.org/i18n.html#pluralization

I learnt it myself recently.